tests/conftest.py

```diff
@@ -188,3 +188,8 @@ def large_query():
         area="POLYGON((0 0,0 10,10 10,10 0,0 0))",
         date=(datetime(2015, 12, 1), datetime(2015, 12, 31)),
     )
+
+
+@pytest.fixture(scope="session")
+def place_kwargs():
+    return ["florida", "ENVELOPE(-87.634896, -79.974306, 31.000968, 24.396308)"]
```

Comment:
I would just this value instead of using a fixture, since it is used only once and adds an unnecessary indirection.